ui/app/mainui/AppMain.qml

```diff
@@ -1589,6 +1595,12 @@ Item {
                         // We should never end up here
                         console.error("AppMain: Unknown section type")
                     }
+                    onCurrentIndexChanged: {
+                        const sectionType = appMain.rootStore.mainModuleInst.activeSection.sectionType
+                        if (sectionType !== Constants.appSection.profile && sectionType !== Constants.appSection.wallet) {
+                            d.maybeDisplayIntroduceYourselfPopup()
```

Comment:
Since `maybeDisplayIntroduceYourselfPopup` is only used once, I'd feel fine moving its code here